public/index.html

```diff
@@ -28,7 +28,15 @@
       catch(e){window.attachEvent("onload", $buo_f)}
     </script>
 
+    <!-- Include Segment Analytics script -->
+    <script>
+      !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t,e){var n=document.createElement("script");n.type="text/javascript";n.async=!0;n.src=("https:"===document.location.protocol?"https://":"http://")+"cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var o=document.getElementsByTagName("script")[0];o.parentNode.insertBefore(n,o);analytics.integrationOptions=e};analytics.SNIPPET_VERSION="4.0.1";
+      analytics.load(%REACT_APP_SEGMENT_ANALYTICS_KEY%);
```

Comment:
This appears as
```js
analytics.load(replace this with segment key);
```
and would be nicer if it appeared as
```js
analytics.load("replace this with segment key");
```
so could you change this to
```js
analytics.load("%REACT_APP_SEGMENT_ANALYTICS_KEY%");
```
?